packages/components/src/components/Button/Button.stories.tsx

```diff
@@ -20,38 +20,55 @@ const meta: Meta<typeof Button> = {
         emphasis: 'default',
         size: 'medium',
         rounding: 'medium',
-        hideLabel: false,
         disabled: false,
         hugWidth: true,
         children: 'Button Text',
+        aspect: 'default',
     },
 };
 export default meta;
 
 export const Default: Story = {
+    name: 'Icon and Text Label',
+    args: {
+        children: (
+            <>
+                <IconColorFan size={16} />
+                Button Text
+            </>
+        ),
+    },
+};
+
+export const TextLabelOnly: Story = {
     name: 'Text Label Only',
 };
 
 export const WithIcon: Story = {
     name: 'Icon Only',
     args: {
-        hideLabel: true,
-        icon: <IconIcon />,
+        icon: <IconIcon size={20} />,
```

Comment:
should be `children` as `icon` will be removed / is deprecated

Reply:
changed